lm_eval/tasks/xnli.py

```diff
@@ -0,0 +1,119 @@
+"""
+XNLI: Evaluating Cross-lingual Sentence Representations
+https://arxiv.org/abs/1809.05053
+
+@InProceedings{conneau2018xnli,
+  author = "Conneau, Alexis
+        and Rinott, Ruty
+        and Lample, Guillaume
+        and Williams, Adina
+        and Bowman, Samuel R.
+        and Schwenk, Holger
+        and Stoyanov, Veselin",
+  title = "XNLI: Evaluating Cross-lingual Sentence Representations",
+  booktitle = "Proceedings of the 2018 Conference on Empirical Methods
+               in Natural Language Processing",
+  year = "2018",
+  publisher = "Association for Computational Linguistics",
+  location = "Brussels, Belgium",
+}
+"""
+
+
+import numpy as np
+from lm_eval.base import rf
+from ..metrics import mean
+from . common import HFTask
+
+
+class XNLIBase(HFTask):
+    VERSION = 0
+    DATASET_PATH = "xnli"
+    DATASET_NAME = None
+
+    def has_training_docs(self):
+        return True
+
+    def has_validation_docs(self):
+        return True
+
+    def has_test_docs(self):
+        return True
+
+    def doc_to_text(self, doc):
+        # following ANLI
+        return doc['premise'] + '\nQuestion: ' + doc['hypothesis'] + ' True, False, or Neither?\nAnswer:'
+
+    def doc_to_target(self, doc):
+        # True = entailment
+        # False = contradiction
+        # Neither = neutral
+        return " " + ["True", "Neither", "False"][doc['label']]
```

Comment:
The few-shot examples in `doc_to_text` have the string "True, Neither, or False?" appended to them, even for non-english languages. Would it be better to translate these and the "Question/Answer" strings there across all the languages? Note we would also have to change these strings in `construct_requests`.

cc @leogao2